IMC_WebDev/MiniCityBlog.md

```diff
@@ -1,31 +1,30 @@
 ## Building a Mini City: HTML Portion
 ## Purpose
-HTML and CSS are extremely important parts of learning about how websites work. Every website uses HTML in order to display things such as text or images on sites. Being able to grasp a few concepts of  HTML and CSS can help a lot when it comes to learning more intricate subjects in computer science. HTML and CSS are tools used to create and edit websites, and it would be nearly impossible to create a website without using at least some HTML/CSS! For this reason, it is important to get some experience with it!
+HTML and CSS are extremely important parts of learning about how websites work. Every website uses HTML in order to display things such as text or images on sites. Being able to grasp a few concepts of  HTML and CSS can help a lot when it comes to learning more intricate subjects in computer science. HTML and CSS are tools used to create and edit websites. For this reason, it is important to get some experience with it!
```

Comment:
Better! But, in the first sentence you say "HTML and CSS are extremely important parts of learning about how websites work." and then later you state "HTML and CSS are tools used to create and edit websites." Aren't these two statements repetitive?